mobile-app/app/components/SubmitButtonGroupV2.tsx

```diff
@@ -76,17 +76,6 @@ export function SubmitButtonGroupV2({
     }
   };
 
-  // const getSubmittingLabel = (): string | undefined => {
```

Comment:
Maybe add comment to describe why

Reply:
Oh I removed it cos it's not used anymore

Reply:
Ops! sorry i thought it was commented